client/src/components/Reviews/components/AddReview.jsx

```diff
@@ -193,6 +201,7 @@ function AddReview({ handleModalToggle, productCharacteristics }) {
               }}
             />
           </div>
+          {submissonErr ? <div>Error in submisson, check all feilds and try again</div> : ''}
```

Comment:
Good to know, i may keep it how it is but that's good to know

Reply:
Yeah I think it's good the way it is. It's something I struggle with because the shorter syntax is less readable but sometimes conditional rendering becomes so messy that having something shorter is nice.